Lib/test/test_os.py

```diff
@@ -1796,36 +1796,46 @@ def test_unicode_name(self):
 
 @unittest.skipIf(sys.platform == "win32", "Posix specific tests")
 class PosixUidGidTests(unittest.TestCase):
+    # uid_t and git_t are 32-bit unsigned integers on Linux
+    UID_OVERFLOW = (1 << 32)
+    GID_OVERFLOW = (1 << 32)
```

Comment:
At 99% this is probably not a real problem in practice, but just in case `uid_t` has a different definition on platforms != Linux or certain architectures, would it make sense to just use a bigger value like `1 << 64`? See:
https://github.com/python/cpython/blob/a24107b04c1277e3c1105f98aff5bfa3a98b33a0/Modules/posixmodule.c#L621

Reply:
I wrote the constants to be able to add a comment and to make the tests more explicit.

I prefer to not change the value before someone reports a bug on a real platform.

Reply:
+1